test/client.test.js

```diff
@@ -4,13 +4,15 @@ import { Strategy } from '../lib/strategy';
 
 import CustomStrategy from './true_custom_strategy';
 import CustomFalseStrategy from './false_custom_strategy';
+import { UnleashEvents } from '../lib/events';
 
-function buildToggle(name, active, strategies, variants = []) {
+function buildToggle(name, active, strategies, variants = [], impressionData= false) {
```

Comment:
Looks like a missing space before `=`. I don't know if prettier is working correctly.